brave/browser-laptop
# Adds UGP support

## Submitter Checklist:


Resolves #11100 

## Test plan
- [x] Plan A:
- clean profile
- check if you see notification about promotion

- [x] Plan B:
- clean profile
- enable payments
- check if first notification is hidden and second one is displayed

- [x] Plan C:
- clean profile
- click on both buttons on the first notification (maybe later, should display first notification after 24h again; second button should open payments page)

- [x] Plan D:
- clean profile
- enable payments
- close preference page
- click on both buttons on the second notification (maybe later, should display second notification after 24h again; second button should open payments page)

- [x] Plan E:
- disable internet connection
- clean profile
- check if payments page have alternative text

- [x] Plan F:
- clean profile
- check if payments page have custom promotion text

- [x] Plan G:
- clean profile
- enable payments 
- check if claim button is there
- click on it
- you would get free BAT's and text below `Add Funds`
- claim button and notifications should be hidden




## Reviewer Checklist:

Tests
